docs/DeveloperGuide.md

```diff
@@ -92,20 +92,43 @@ The `UI` component:
 
 ![Structure of the Logic Component](images/LogicClassDiagram.png)
 
-**API** :
-[`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
+<div markdown="block" class="alert alert-info"> 
+:information_source: **Note:** ABC refers to `Deliverable`, `Meeting`, `Person`. 
+XYZ refers to any command call, e.g Add, find, delete, etc. 
+
+</div>
+
+**API** : <br>
+* [`LogicDeliverable.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicDeliverable.java)
+* [`LogicMeeting.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicMeeting.java)
+* [`LogicPerson.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicPerson.java)
+* [`LogicDispatcher.java`](https://github.com/AY2021S1-CS2103T-F11-2/tp/tree/master/src/main/java/seedu/address/logic/LogicDispatcher.java)
+
+The `Logic` component parses the user commands and executes them. There are different pa`LogicDispatcher` selects the correct parser to 
+
+This is the list of what `Model` components affected by their respective `Logic` components: 
+* `LogicDeliverable`: Component that affects `DeliverableModel`.
+* `LogicMeeting`: Component that affects `MeetingModel`.
+* `LogicPerson`: Component that affects `PersonModel`.
 
-1. `Logic` uses the `AddressBookParser` class to parse the user command.
-1. This results in a `Command` object which is executed by the `LogicManager`.
+The components follow the general sequence to execute a command:
+
+1. `Logic` uses the `XYZBookParser` class to parse the user command.
+1. This results in a `XYZCommand` object which is executed by the `LogicXYZManager`.
 1. The command execution can affect the `Model` (e.g. adding a person).
 1. The result of the command execution is encapsulated as a `CommandResult` object which is passed back to the `Ui`.
 1. In addition, the `CommandResult` object can also instruct the `Ui` to perform certain actions, such as displaying help to the user.
 
-Given below is the Sequence Diagram for interactions within the `Logic` component for the `execute("delete 1")` API call.
+Given below is the Sequence Diagram for interactions within the `Logic` component for API call of any command. 
 
-![Interactions Inside the Logic Component for the `delete 1` Command](images/DeleteSequenceDiagram.png)
+![Interactions Inside the Logic Component for any command](images/CommandSequenceDiagram.png)
```

Comment:
Remove the color for outer box 
![Screenshot 2020-11-09 at 4 02 06 PM](https://user-images.githubusercontent.com/52078276/98514735-ebbdd680-22a4-11eb-9bc5-4893745b0ece.png)

Reply:
- Add XYZ note 
- I think execute("XYZ args") shouldnt be XYZ args. If any command then can just put execute(command).
- there is a d in d:XYZCommand --> not sure what is it and if its meaningful
- i think the condition in opt can be "command affects models"

Reply:
XYZ is meant to represent the the type of command
d is meant to represent the object, will change to `abcCommand` instead
Will amend accordingly